UX/ui-service/MyOrders/design.md

```diff
@@ -0,0 +1,42 @@
+# My Orders
+
+- My Orders is a primary navigation item
+- Administrators
+  - View and approve or deny orders for other users
+- End Users
+  - View status of placed orders
+
+## Admin user
+
+### Review Orders
+![Image Title/Mockup Name](img/Order_01.png)
```

Comment:
@cshinn I think that the states should be Pending, Approved or Denied.   Can you update the mocks?